mesonbuild/interpreter.py

```diff
@@ -354,23 +354,27 @@ class BuildMachine(InterpreterObject):
     def __init__(self, compilers):
         self.compilers = compilers
         InterpreterObject.__init__(self)
+        self.held_object = environment.MachineInfo(environment.detect_system(),
+                                                   environment.detect_cpu_family(self.compilers),
+                                                   environment.detect_cpu(self.compilers),
+                                                   sys.byteorder)
         self.methods.update({'system': self.system_method,
                              'cpu_family': self.cpu_family_method,
                              'cpu': self.cpu_method,
                              'endian': self.endian_method,
-                             })
+                            })
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->